confobo/models/calendar.py

```diff
@@ -0,0 +1,35 @@
+import os
+from icalendar import Calendar as ICalendar
+
+from .event import Event
+from confobo.config import PROJECT_ROOT
+
+SAMPLE_FILE = os.path.join(PROJECT_ROOT, 'data/calendar.ics')
```

Comment:
I don't know. Right now `data/calendar.ics` is a “real” schedule and there's another one in `tests/data/calendar.ics` created for tests.
I didn't try to come up with a better way, guess it doesn't matter much now. It'll be reorganized a dozen times anyway.